app/presenters/supergroups/supergroup.rb

```diff
@@ -47,6 +42,15 @@ def data_module_label
 
   private
 
+    def finder_path
+      "/search/#{name.dasherize}"
+    end
+
+    def finder_url(base_path)
+      query_string = { parent: base_path }.to_query
```

Comment:
This should set the `level_one_taxon` and `level_two_taxon` parameters - though I'm not actually sure `level_two_taxon` is required, eg https://www-origin.publishing.service.gov.uk/search/news-and-communications?level_one_taxon=17fb491a-15f3-4c71-8e25-f645ff8b3ed3 works and isn't a level one taxon

Reply:
This should actually set the new `topic` parameter @koetsier is adding: https://trello.com/c/rjrtLu4N/821-allow-topic-parameter-in-supergroup-finder

Reply:
Thanks, changed.